src/Core/Framework/Resources/config/packages/framework.yaml

```diff
@@ -35,7 +35,9 @@ framework:
     mailer:
         message_bus: false
         dsn: '%env(MAILER_DSN)%'
-    lock: '%env(LOCK_DSN)%'
+    lock:
+        default: '%env(LOCK_DSN)%'
+        doctrine: 'Doctrine\DBAL\Connection'
```

Comment:
do we need to use doctrine lock in this case? if so we can adjust the comment in the verifier ;)

Reply:
Doctrine is guaranteed to expire, so thats why we use it